definitions/divorce/json/SearchCasesResultFields/SearchCasesResultFields-resp-journey-roles-and-permissions-nonprod.json

```diff
@@ -0,0 +1,80 @@
+[
+  {
```

Comment:
Those are the columns the caseworker-caa will see when viewing the list of unassigned cases